docs/guides/accessing-the-dom.md

```diff
@@ -25,70 +25,120 @@ class MyComponent extends React.Component {
+
+- ```js
+  class GoodComponent extends React.Component {
+    constructor(props) {
+      super(props)
+      this.ref = React.createRef()
+    }
+    render() {
+      return <span ref={this.ref}>Good Position target component</span>
+    }
   }
-  render() {
-    return (<span ref={this.ref}>Good Position target component</span>)
+  const Example = () => {
+    return (
+      <View as="div" padding={'large'}>
+        <Position
+          renderTarget={<GoodComponent />}
+          placement="end center"
+          offsetX="20px"
+        >
+          <span style={{ padding: '8px', background: 'white' }}>
+            Positioned content
+          </span>
+        </Position>
+      </View>
+    )
   }
-}
+  render(<Example />)
+  ```
 
-const Example = () => {
-  return (
-    <View as="div" padding={'large'}>
-      <Position
-        renderTarget={<GoodComponent />}
-        placement='end center'
-        offsetX='20px'
-      >
-        <span style={{ padding: '8px', background: 'white' }}>Positioned content</span>
-      </Position>
-    </View>
-  )
-}
+- ```js
+  const GoodComponent = React.forwardRef((props, ref) => {
+    return <span ref={ref}>Good Position target component</span>
+  })
 
-render(<Example />)
-```
+  const Example = () => {
+    return (
+      <View as="div" padding={'large'}>
+        <Position
+          renderTarget={<GoodComponent />}
+          placement="end center"
+          offsetX="20px"
+        >
+          <span style={{ padding: '8px', background: 'white' }}>
+            Positioned content
+          </span>
+        </Position>
+      </View>
+    )
+  }
+  render(<Example />)
+  ```
 
 Bad Usage Example without `ref`, that will result in InstUI calling `ReactDOM.findDOMNode()` and throw warnings:
 
-```js
----
-type: example
----
-class BadComponent extends React.Component {
-  constructor(props) {
-    super(props)
+- ```js
+  class BadComponent extends React.Component {
+    constructor(props) {
+      super(props)
+    }
+    render() {
+      return <span>Bad Position target component</span>
+    }
   }
-  render() {
-    return (<span>Bad Position target component</span>)
+
+  const Example = () => {
+    return (
+      <View as="div" padding={'large'}>
+        <Position
+          renderTarget={<BadComponent />}
+          placement="end center"
+          offsetX="20px"
+        >
+          <span style={{ padding: '8px', background: 'white' }}>
+            Positioned content
+          </span>
+        </Position>
+      </View>
+    )
   }
-}
+  render(<Example />)
+  ```
 
-const Example = () => {
-  return (
-    <View as="div" padding={'large'}>
-      <Position
-        renderTarget={<BadComponent />}
-        placement='end center'
-        offsetX='20px'
-      >
-        <span style={{ padding: '8px', background: 'white' }}>Positioned content</span>
-      </Position>
-    </View>
-  )
-}
+- ```js
+  const BadComponent = React.forwardRef((props, ref) => {
```

Comment:
this example doesn't display the positioned component on the docs page

Reply:
The example is purposefully bad usage example because it's without ref. Unfortunately I couldn't find a way to write a "bad example" without ref that displays the positioned component. Any suggestion?

Reply:
I see, in that case I'd might remove that example all together and present it in a different way. Maybe just the code itself or something like that.

Reply:
I converted the bad examples to just a code.

Reply:
I have converted the bad examples back to the Class/Function example format